### ISSUE TITLE
When you visit light theme website with dark mode its not looking correct

### ISSUE THREAD

**Issue by sametozturk45** [zen-browser/desktop#3637]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

When i visit light theme website, its not looking correctly.When I add background-color:#fff in to body of html manualy its fixing. 

Some example websites (You should test with your browser active theme is dark):
- [BoxIcon](https://boxicons.com/)
- [Udemy](https://www.udemy.com/)

Boxicon (Zen Browser) : 
![{FF85A415-E58E-4F95-9B5F-CC66FF755E82}](https://github.com/user-attachments/assets/3cd87f28-c7d1-48c2-8f40-fb95c26d49e8)

Boxicon (Other Browser) : 
![{54027778-ECA3-41B8-8FB7-4C84DC7EABA6}](https://github.com/user-attachments/assets/1a1a48e2-cfef-4b9a-b76c-bf88238f8f10)

Udemy (Zen Browser) : 
![{79276EF6-B46B-437B-AC3C-D7E8CB9C2603}](https://github.com/user-attachments/assets/42debc91-426d-4865-8b87-6e4edc5a4d19)

Udemy (Other Browser) : 
![{747C22AB-4B4F-4E1B-A3C8-8DEF5E3BB0BF}](https://github.com/user-attachments/assets/fe5e8e03-c97a-4f01-a6d6-8c00f5adc338)

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.2-b.0

### What platform are you seeing the problem on?

Windows

**Comment by Likhith-18** [comment]
Yes same issue i created the same issue still didnt resolve it please fix it same in 1.0.2-b.1 also

**Comment by LautiLosio** [comment]
+1 to this
I've been "fixing" this by using the darkreader extension, but I hope this gets looked into soon!

**Comment by jurienhamaker** [comment]
Seems to h appen when websites don't have a default background on the body, resulting in the browser's viewport's background. which in zen's case is transparent?

**Comment by JSGRD22** [comment]
Set `browser.tabs.allow_transparent_browser` to `false` in about:config

**Comment by LautiLosio** [comment]
> Set `browser.tabs.allow_transparent_browser` to `false` in about:config

Great workaround! Would love to know how to get this properly working tho.

**Comment by BoscoDomingo** [comment]
Same here. All websites also switch from Dark to Light, even if I've selected Dark mode explicitly. Best example is ChatGPT, where all the fonts are in white (as if in dark mode) but the background is also white, which makes it impossible to read


https://github.com/user-attachments/assets/56f3b7b3-201b-4439-b84d-f092ca69b104

**Comment by tobias-hay** [comment]
> Same here. All websites also switch from Dark to Light, even if I've selected Dark mode explicitly. Best example is ChatGPT, where all the fonts are in white (as if in dark mode) but the background is also white, which makes it impossible to read
> Recording.2024-12-19.194407.mp4

These are not the same bug

**Comment by mr-cheffy** [comment]
Please, search for duplicates, please

https://github.com/zen-browser/desktop/issues/3295
